modules/yieldlabBidAdapter.js

```diff
@@ -186,13 +186,21 @@ export const spec = {
           // there may be publishers still rely on it
```

Comment:
This is also rather a cosmetic suggestion but as we access `matchedBid.native` and `matchedBid.native.assets` quite a few times, we could fetch them once from the `matchedBid` object and directly use them later which cleans the calling places up a bit.

```suggestion
          const { native } = matchedBid;
          const { assets } = native;

          // there may be publishers still rely on it
```

So with this suggestion and the comment below we might transform

```
const nativeIconAssetObj = find(matchedBid.native.assets, asset => isImageAssetOfType(asset, 1));
```

to something like

```
const iconAssetObj = find(assets, iconAsset);
```